MAINTAINERS.md

```diff
@@ -205,8 +205,13 @@ Here's an example of RFC with good engagement from the community: [aws-powertool
 
 ### Releasing a new version
 
-> [!note]
-> This section is a work in progress and will be updated as the project evolves - see [#57](https://github.com/aws-powertools/powertools-mcp/issues/57)
+Whenever a new version is ready to be released, follow these steps:
+
+1. Run the **Make Version** workflow to bump the version in the `package.json` file and update teh `CHANGELOG.md` file. This will create a new commit and open a pull request with the changes.
```

Comment:
```suggestion
1. Run the **Make Version** workflow to bump the version in the `package.json` file and update the `CHANGELOG.md` file. This will create a new commit and open a pull request with the changes.
```